### PR TITLE
Add back pino-related typings on IncomingMessage, OutgoingMessage and…

### PR DESCRIPTION
**Pull request by oddeirik**
… ServerResponse.

Also fixed a test that fails on Windows (no /dev/null available, will print to STDOUT instead).

Ref. #175.
These additional typings are included in the @types/pino-http, so I just added them back in.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for pino-http 6.0
+// Type definitions for pino-http 6.x
```
Comment (mcollina) [inline index.d.ts:1]: Let's remove this line

**Review by mcollina — COMMENTED** [review]
@kibertoad can you review?

**Review by kibertoad — APPROVED** [review]

**Review by mcollina — APPROVED** [review]
lgtm
